tests/cli/test_compose_live-iso.sh

```diff
@@ -13,8 +13,6 @@ set -e
 . $(dirname $0)/lib/lib.sh
 
 CLI="${CLI:-./src/bin/composer-cli}"
-QEMU_BIN="/usr/bin/qemu-system-$(uname -m)"
-QEMU="$QEMU_BIN -machine accel=kvm:tcg"
```

Comment:
OK, I fixed that. However, is this still needed [in general], as we moved to using the separate function for starting the VM? Btw, now that I looked again at the cherry-picked commit, is `export QEMU="/usr/libexec/qemu-kvm"` what we really want on Fedora? Unforntunately I haven't been able yet to go past the installation step in this test on Fedora (bugs during installation on F-30, issues with packages downloading on rawhide), so that's probably why I haven't noticed this sooner.

Reply:
I don't see this to have changed. On Fedora test systems there's `qemu-system-x86_64` but no `qemu-kvm` so the qcow2 test fails.